.github/ISSUE_TEMPLATE/2_new-hub-collect-info.yml

```diff
@@ -65,42 +37,82 @@ body:
       description: |
         What kind of authentication service will be used for this community. This is the type of "log-in username" users will use on the hub.
       options:
-        - GitHub Authentication (e.g., @mygithubhandle)
-        - Google OAuth (myemailaddress@gmail.org, or a Google-backed domain)
-        - CILogon (e.g., username@institution.org)
+        - GitHub (e.g., @mygithubhandle)
+        - Google (myemailaddress@gmail.org, or a Google-backed domain)
+        - Institutional Provider via CILogon (e.g., username@institution.org)
         - Username / Password via auth0: https://auth0.com/docs/connections/database
```

Comment:
This line doesn't render nicely :/

<img width="316" alt="Screenshot 2022-10-07 at 13 54 22" src="https://user-images.githubusercontent.com/44771837/194558897-c9c62eef-75c0-4434-8dc1-122870980eab.png">

Reply:
I think the `:` were to blame 🙄 Thanks!